### DISCUSSION TITLE
Bottom keyboard bar covering the terminal window

### DISCUSSION THREAD

**Discussion by djsnipa1** [blinksh/blink#1737] [Support] [answered]
I am unable to use Blink because when I use my external keyboard, the bottom of the terminal window gets covered by a keyboard bar. I’m using an iPhone SE with the latest version of Blink. Thanks for your help!

Here is my screen with the on screen keyboard. Notice the tmux status bar right above it. That is how I need it to look when I connect my external keyboard.
![B5711AC2-AC6E-40EC-94B3-D42FF6B39D05](https://user-images.githubusercontent.com/118037/228204503-c6b76084-1b7c-4d91-a191-ea8be0b535a8.png)

Here is what I’m trying to solve. The tmux status bar is completely covered. I’m unable to use neovim as well because of this.
![611E45D1-94F4-4B6D-9A4C-589DBA1BDEBF](https://user-images.githubusercontent.com/118037/228204857-856968ec-7a8c-4231-aa08-a3d4f9ac680c.png)

**Comment by carloscabanero** [comment, marked as answer]
Sorry about that. This is a bug. I think there was some reason why with iOS we could not get rid of the bar, but the calculation is all wrong anyway. Will try to fix this week.

**Reply by carloscabanero** [reply]
This is now available on TestFlight 747, https://github.com/blinksh/blink/issues/1738

**Reply by djsnipa1** [reply]
Thank you so much! It works perfectly now and is my iOS shell of choice!
